chrome/content/zotero/itemTreeColumns.jsx

```diff
@@ -23,34 +23,36 @@
 	***** END LICENSE BLOCK *****
 */
 
-(function() {
 const React = require('react');
 const Icons = require('components/icons');
 
 /**
- * @type Column {
- * 	dataKey: string,				// Required, see use in ItemTree#_getRowData()
- *
- * 	defaultIn: Set<string>,			// Types of trees the column is default in. Can be [default, feed];
- * 	disabledIn: Set<string>,		// Types of trees where the column is not available
- * 	defaultSort: number				// Default: 1. -1 for descending sort
- *
- * 	flex: number,					// Default: 1. When the column is added to the tree how much space it should occupy as a flex ratio
- * 	width: string,					// A column width instead of flex ratio. See above.
- * 	fixedWidth: boolean				// Default: false. Set to true to disable column resizing
- * 	staticWidth: boolean			// Default: false. Set to true to prevent columns from changing width when
- * 									// the width of the tree increases or decreases
- * 	minWidth: number,				// Override the default [20px] column min-width for resizing
- *
- * 	label: string,					// The column label. Either a string or the id to an i18n string.
- * 	iconLabel: React.Component,		// Set an Icon label instead of a text-based one
- *
- * 	ignoreInColumnPicker: boolean	// Default: false. Set to true to not display in column picker.
- * 	submenu: boolean,				// Default: false. Set to true to display the column in "More Columns" submenu of column picker.
- *
- * 	primary: boolean,				// Should only be one column at the time. Title is the primary column
- * 	zoteroPersist: Set<string>,		// Which column properties should be persisted between zotero close
- * 	}
+ * @typedef ItemTreeColumnOption
+ * @type {object}
+ * @property {string} dataKey - Required, see use in ItemTree#_getRowData()
+ * @property {Set.<string>} [defaultIn] - Types of trees the column is default in. Can be [default, feed];
+ * @property {Set.<string>} [disabledIn] - Types of trees where the column is not available
```

Comment:
I guess this is from the original, but since this will be exposed now, I think it should just be `{String[]}` for these and for `zoteroPersist` — requiring `Set` is kind of annoying. We generally don't even do it internally elsewhere in the code. The receiving function can always make the Set.

(Also, are you getting the period from somewhere specific? We always specify these as `Set<String>`, etc.)